sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/TextAnalyticsClientJavaDocCodeSnippets.java

```diff
@@ -274,17 +276,31 @@ public void recognizePiiEntitiesWithLanguage() {
     }
 
     /**
-     * Code snippet for {@link TextAnalyticsClient#recognizePiiEntitiesBatch(Iterable, String, TextAnalyticsRequestOptions)}
+     * Code snippet for {@link TextAnalyticsClient#recognizePiiEntities(String, String, RecognizePiiEntityOptions)}
+     */
+    public void recognizePiiEntitiesWithRecognizePiiEntityOptions() {
+        // BEGIN: com.azure.ai.textanalytics.TextAnalyticsClient.recognizePiiEntities#String-String-RecognizePiiEntityOptions
+        textAnalyticsClient.recognizePiiEntities("My SSN is 859-98-0987", "en",
+            new RecognizePiiEntityOptions().setDomainFilter(PiiEntityDomainType.PROTECTED_HEALTH_INFORMATION))
+            .forEach(entity -> System.out.printf(
+                "Recognized Personally Identifiable Information entity: %s, entity category: %s,"
+                    + " entity subcategory: %s, confidence score: %f.%n",
+                entity.getText(), entity.getCategory(), entity.getSubcategory(), entity.getConfidenceScore()));
+        // END: com.azure.ai.textanalytics.TextAnalyticsClient.recognizePiiEntities#String-String-RecognizePiiEntityOptions
```

Comment:
Can you add a test for domain filter where a returned entity gets filtered out? [Here](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/textanalytics/azure-ai-textanalytics/tests/test_recognize_pii_entities.py#L599) is a python test for taht

Reply:
There are a few tests for domain filter already in the PR. You are looking at the codesnippet section. https://github.com/Azure/azure-sdk-for-java/pull/14714/files#diff-9df5b3d7d96dd8e2313b619beb609285R509